tools/internal_ci/linux/grpc_bloat_diff_in_docker.sh

```diff
@@ -24,3 +24,8 @@ time python3 -m pip install --user -r tools/internal_ci/helper_scripts/requireme
 
 tools/internal_ci/linux/run_if_c_cpp_modified.sh tools/profiling/bloat/bloat_diff.py \
   -d "origin/$KOKORO_GITHUB_PULL_REQUEST_TARGET_BRANCH"
+
+tools/run_tests/start_port_server.py
+
+tools/internal_ci/linux/run_if_c_cpp_modified.sh tools/profiling/memory/memory_diff.py \
```

Comment:
as always with two subsequent tasks in the same job, this is a bit hacky. You probably need to use ` || FAILED=true` and `if [ "$FAILED" != "" ]` at the end of the script to make sure these two scripts are at least a bit independent (the second one runs if the first one fails)

Should there be a completely separate job for memory_diff?



e.g. here:
https://github.com/grpc/grpc/blob/b0cdd3cba29928da3652bded596a64bda68c75c1/tools/internal_ci/linux/grpc_distribtests_csharp.sh#L64
https://github.com/grpc/grpc/blob/b0cdd3cba29928da3652bded596a64bda68c75c1/tools/internal_ci/linux/grpc_distribtests_csharp.sh#L35

Reply:
i think fixed with the split up of bloat/memory jobs